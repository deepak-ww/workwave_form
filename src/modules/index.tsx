export * from './auth/login';
export * from './requestForm/requestForm';
export * from './error/errorBoundary';