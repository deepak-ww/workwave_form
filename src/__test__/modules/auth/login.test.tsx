import React from 'react';
import { configure, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { Login } from '../../../modules';

configure({ adapter: new Adapter() });

describe("Login", () => {
    let login = mount(<Login />);

    it('Render Login component', () => {
        expect(login).toMatchSnapshot();
    })

    it('Validate Login title', () => {
        console.log(login.debug());
        expect(login.find('.logo').text()).toEqual('WorkWave Form');
    })

    it('Login user & password field exits', () => {
        expect(login.find('#login-username').exists()).toBe(true);
        expect(login.find('#login-password').exists()).toBe(true);
    });

    it('Render the login button', () => {
        expect(login.find('.sc-bdVaJa').text()).toEqual('Login');
    });
});