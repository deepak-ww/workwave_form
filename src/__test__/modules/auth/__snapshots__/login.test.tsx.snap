// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Render Login component 1`] = `ReactWrapper {}`;
